src/pages/Login.vue

```diff
@@ -434,6 +434,7 @@ export default {
         let socialAuthToken = googleUser.getAuthResponse();
         UserAPIService.convertSocialAuthToken(socialAuthToken.access_token).then(
           (response) => {
+            console.log('response', response)

```

Comment:
Will you remove this later on, before merging?

Reply:
Thanks for catching it. Removed!